src/main/java/com/google/sps/data/User.java

```diff
@@ -17,17 +17,21 @@
 /** The class represents user details. */
 public final class User {
 
-  private final String userEmail;
+  private final String email;
   private final String name;
   private final String department;
+  private final int year;
+  private final long phone;
```

Comment:
Would it be better to have it as a string? To allow leading zeros for example

Reply:
great idea, thank you for the suggestion :D